apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/McsClaimTransformerV2.java

```diff
@@ -31,13 +31,17 @@
 import org.hl7.fhir.r4.model.Resource;
 import org.hl7.fhir.r4.model.codesystems.ClaimType;
 import org.hl7.fhir.r4.model.codesystems.ProcessPriority;
+import org.springframework.stereotype.Component;
 
 /** Transforms FISS/MCS instances into FHIR {@link Claim} resources. */
+@Component
 public class McsClaimTransformerV2 extends AbstractTransformerV2 {
 
   /** The metric name. */
-  private static final String METRIC_NAME =
-      MetricRegistry.name(McsClaimTransformerV2.class.getSimpleName(), "transform");
+  private final String METRIC_NAME;
+
```

Comment:
Is there a reason to turn a constant into an instance variable?

Reply:
Didn't notice at the time this was just grabbing the class name when I was replacing all the static method calls, so no. Reverted to a constant.